README.md

```diff
@@ -65,26 +65,44 @@ The supported flags are described below:
 
 ```
 $ ./monaco
-Please provide environments yaml with -e/--environments!
+Name:
+        monaco
+Version:
+        1.0.1
+
+Usage:
```

Comment:
Do we actually really want to duplicate the commands' help output in the readme? 

This will need to be pasted over whenever the tool changes, and that seems easy to forget and get out of sync eventually...

Reply:
do you then suggest to remove that part of the documentation completely ? or just list the flags only ?

Reply:
I'd suggest to remove the full verbatim copy, and just leave some example usages. 
but what's your opinion @CruzanCaramele & @didiladi ?

Reply:
Fine by me. As long as this one is in there (posted above):

```
Usage:
   monaco --environments <path-to-environment-yaml-file> [--specific-environment <environment-name>] [--project <project-folder>] [--dry-run] [--verbose] [projects-root-folder]
```

Its a concise description of:
* which option is optional or required 
* and that the folder needs to be the last argument

Reply:
I will include the examples and what @didiladi mentioned above